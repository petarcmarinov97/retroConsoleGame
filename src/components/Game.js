import React, { Component } from 'react';
import "../styles/Game.css";
import Tetris from './Tetris';

export default class Game extends Component {

    render() {
        return <Tetris />
    }
}
