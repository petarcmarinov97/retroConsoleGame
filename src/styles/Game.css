canvas {
    margin: 4px 1px 0px 4px;
    max-height: 400px;
    max-width: 200px;
    height: 400px;
    width: 200px;
    border: 1px solid black;
}

div#paused {
    margin: 4px 1px 0px 4px;
    height: 400px;
    width: 200px;
    border: 1px solid black;
}

div#gameOver {
    text-align: center;
    margin: 4px 1px 0px 4px;
    height: 400px;
    width: 200px;
    border: 1px solid black;
}