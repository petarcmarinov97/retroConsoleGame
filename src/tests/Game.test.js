import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import Game from '../components/Game';

describe('Game tests', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Game />);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})