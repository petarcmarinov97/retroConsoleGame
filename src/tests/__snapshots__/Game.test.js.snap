// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game tests Snapshots should render correctly 1`] = `<Tetris />`;
